### ISSUE TITLE
[Bug]: [P1] Common accessibility issue - First element read by Voice Over should be the page title

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ods-ios#572]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Plus

### OS Version

17.1.1

### App version

v0.16.0

### Describe the issue

The navigation with Voice Over is not consistent from a component to another.
For many component and variants, when the page is displayed then the first element read by Voice Over is the "customize" label (example : Card or Bars-navigation variants).
On some other components or variants, the first element read is the content view (like Banners or Bars-tabs variant).
This is not consistent.

### Expected Behavior

For every components and variants pages, the first element read by Voice Over should be the title of the page. So the user is informed about the context of the page.

**Comment by pya35** [comment]
Seems to be ok now

**Comment by ludovic35** [comment]
@pya35 Ça you confirm everything it is ok, now, please  ?
